src/Utils/JsFunctionsScanner.php

```diff
@@ -37,11 +37,18 @@ public function getFunctions(array $constants = [])
 
             switch ($char) {
                 case '\\':
-                    $prev = $char;
-                    $char = $next;
                     $pos++;
+
+                    if ($next !== 'n') {
```

Comment:
The escaped character cannot be only `\n` but any character (like, for example, `\t`).
I think we should accept any character inside double or single quotes.

Reply:
I'll look into it. So essentially we could just remove the whole `\\` case from the switch statement?

Reply:
I guess just adding something like this:

```php
switch ($this->status()) {
    case 'simple-quote':
    case 'double-quote':
        break;
    default:
        //everything else
}
```